### PR TITLE
HTTP to ServiceNow - Incident

### LINKED ISSUE #155: HTTP to ServiceNow - Create incident
1. Publish a HTTP endpoint
2. Endpoint will provide a JSON payload
3. Create incident in ServiceNow about the incoming incident

### PR DESCRIPTION
**Pull request by jannejjj**
closes #155 

Checklist for review:
- [x] Process name is fine, uses normal casing and descriptive
  - [x] If applicable, the process name follows the naming convention "{source/format} to {destination/format} - {object}"
- [x] Process description is fine, uses normal casing, dots and is descriptive
  - [x] Element names use normal casing, no unneeded capital letters
- [x] Correct casing used for formats and terms, e.g. CSV, FTP, JSON, Avro
  - [x] Element names do not contain dots
  - [x] Comments / note element text are in good language and end with dots
- [x] Template parameters checked
  - [x] Parameter follows the CamelCase naming convention, abbreviations included, e.g. ConnectionString, BaseUrl, ApiKey
  - [x] Parameter descriptions are good and end with dots
  - [x] Sensitive parameters are marked as secrets
- [x] "Turn off logging" switch is on for any task that has sensitive data as output, e.g. tokens in the header.
- [x] Process end points like error throws and return statements have titles (double-click) and reasonable information.
  - [x] Results have good summaries, preferably with statistics of some sort like "processed X files"
  - [x] Errors have some error info, usually from task (e.g. some ErrorMessage property contains additional error info)
  - [x] Errors are pointed downwards from their respective source, e.g. an exclusive decision
- [x] Process is organised with groups (dashed boxes)
- [x] Straight lines are preferred where possible
- [x] Subprocesses are forbidden
- [x] Text mode is preferred over Expressions in expressions, e.g. instead of *$"This is {#var.error}"* use *This is {{#var.error}}*
- [x] Process metadata file is present and content is fine (version number and name)
- [x] Process export file is present (process.json)
- [x] Process description file is present (all sections filled in)
- [x] Process description has a SVG screenshot of the process
- [x] Template files are going to the correct folder
- [x] Markdown paragraphs are separated with two newlines, so there is a blank line inbetween two paragraphs
- [x] If some system is integrated using e.g. a HTTP task (meaning we do not use a native connector for the system), then the system should be mentioned in additional systems used in metadata.json
- [x] If the template is handling some sample JSON data - make sure that the sample data is included in the long description (just one-two elements is enough)
- [x] If the template is assuming that there is a database table ready for data insertion / updates - include the table structure in the long description as a `CREATE TABLE` scipt.

## Summary by CodeRabbit

- **New Features**
  - Introduced a template for processing HTTP POST requests to create incidents in ServiceNow, streamlining the incident management process.
  - Created a structured process template for handling incoming incidents via HTTP requests, including detailed error handling and response outcomes.

- **Documentation**
  - Provided comprehensive guidelines on prerequisites, request formats, and error handling for the new incident creation template.
  - Added metadata for integration management and version control in the new ServiceNow incident template.

### CODE REVIEW

File: Templates/Service Management/ServiceNow/HTTP to ServiceNow- Indicent/long-description.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+This template process receives an HTTP POST request containing incident information, and creates an incident into ServiceNow based on it.
+
+![Template](assets/HTTP-to-ServiceNow-Create-incident.svg)
+
+# Prerequisites
+
+This template assumes that the following prerequisites are in place:
+
+- The ServiceNow user has the required permissions to create incidents and modify their fields.
+- - The default role required for this in ServiceNow is "sn_incident_write".
```
[the remark this answers is not in the document]
Comment (jannejjj) [reply Templates/Service Management/ServiceNow/HTTP to ServiceNow- Indicent/long-description.md:10]: Thanks coderabbit!

File: Templates/Service Management/ServiceNow/HTTP to ServiceNow- Indicent/long-description.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+```JSON
+{
+    "number": "INC00001",
+    "id": "abcdefg1234567"
+}
+
+{
+    "error": "An exception occurred when attempting to create the incident into ServiceNow. HTTP status code: 401"
+}
+```
```
[the remark this answers is not in the document]
Comment (jannejjj) [reply Templates/Service Management/ServiceNow/HTTP to ServiceNow- Indicent/long-description.md:47]: I think we've used the bolding before for things like this so i'll leave this as it is.

File: Templates/Service Management/ServiceNow/HTTP to ServiceNow- Indicent/long-description.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+This template process receives an HTTP POST request containing incident information, and creates an incident into ServiceNow based on it.
+
+![Template](assets/HTTP-to-ServiceNow-Create-incident.svg)
+
+# Prerequisites
+
+This template assumes that the following prerequisites are in place:
+
+- The ServiceNow user has the required permissions to create incidents and modify their fields.
+  - The default role required for this in ServiceNow is "sn_incident_write".
```
[the remark this answers is not in the document]
Comment (charlotta-frends) [reply Templates/Service Management/ServiceNow/HTTP to ServiceNow- Indicent/long-description.md:10]: I feel that the two sentences could be joined as just one bullet point.

File: Templates/Service Management/ServiceNow/HTTP to ServiceNow- Indicent/long-description.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
+This template process receives an HTTP POST request containing incident information, and creates an incident into ServiceNow based on it.
```
Comment (charlotta-frends) [inline Templates/Service Management/ServiceNow/HTTP to ServiceNow- Indicent/long-description.md:1]: Up for preference, but could something like "...and creates a corresponding incident in ServiceNow." be nicer?

**Review by charlotta-frends — COMMENTED** [review]
Works nicely, just a few things suggested. Also, check spelling of the folder name (missing space before -). Apart from that, everything's looking good!

**Review by charlotta-frends — APPROVED** [review]

**Pull request merged**